backend/server/api/services/todos.service.js

```diff
@@ -18,5 +18,19 @@ export class ToDoService {
     const newTodo = allTodos.child(newTodoRef.key).toJSON();
     return { ...newTodo, key: newTodoRef.key };
   }
+  async deleteByID(key) {
+    var ref = this.db.ref("todo/" + key);
+    var success = await ref.once("value").then(function (snapshot) {
```

Comment:
You can omit the `.then()` and just directly put its body. Also by doing that, you wouldn't need to use a `success` variable